viewing_party/party.py

```diff
@@ -1,23 +1,136 @@
 # ------------- WAVE 1 --------------------
 
+
+
+
+from webbrowser import get
+
+
 def create_movie(title, genre, rating):
-    pass
+    movie = None
+    if title == None or genre == None or rating == None:
+        movie == None
+    else:
+        movie = {
+            "title" : title,
+            "genre" : genre,
+            "rating" : rating
+        }
+    return movie

```

Comment:
On L12, we check if `movie` is `None`, not reassigning movie as `None`. We can also get rid of the temporary variable and return directly inside the else loop, like so:
```
if title == None or genre == None or rating == None:
    return None
else:
    return = {
        "title": title,
        "genre": genre,
        "rating": rating
    }
```